jenkins_execute_cmds.txt

```diff
@@ -1,30 +0,0 @@
-#!/bin/bash
```

Comment:
Ancient cruft, the test file still refers to users and passwords, which haven't worked since 2016 or so